app/common/update.py

```diff
@@ -253,3 +258,65 @@ def merge_local_db():
 
         logger.info(str(records_inserted) + " records were inserted into local db.")
         logger.info(str(records_updated) + " records in local db were updated.")
+        
+
+def download_dat_files():
+    config = load_user_config()
+    valid_directory = False

```

Comment:
Instead of using the `valid_directory` False/True concept, we could just check what's in the config on line 296 and act on that.

```python
if os.path.exists(config["config"]["installdirectory"]):
  ...
```